RecFCCeeCalorimeter/src/components/AugmentClustersFCCee.cpp

```diff
@@ -120,38 +163,48 @@ StatusCode AugmentClustersFCCee::execute([[maybe_unused]] const EventContext &ev
     sumEnLayer.assign(numLayersTotal, 0.);
     double phiMin = 9999.;
     double phiMax = -9999.;
+    int module_id_Min = 1536;
+    int module_id_Max = -1;
 
     // loop over each system/readout
     int startPositionToFill = 0;
     for (size_t k = 0; k < m_readoutNames.size(); k++)
     {
       if (k > 0)
       {
-        startPositionToFill += m_numLayers[k - 1];
+        startPositionToFill += m_numLayers[k - 1];  // only 2 sub detectors ?
```

Comment:
these shower shape variables for photon/pi0 discrimination are only useful for the ECAL.. does it make sense to compute and save them also for the HCAL?

Reply:
I would probably at least add a boolean flag to calculate or not these pi0/gamma shower shapes so that the user can turn them on only for ECAL-only clusters